virttest/virt_admin.py

```diff
@@ -702,6 +704,25 @@ def command(cmd, **dargs):
     return ret
 
 
+def check_server_name(server_name="virtproxyd"):
+    """
+    Determine the server name under different daemon mode.
+    """
+    if not utils_split_daemons.is_modular_daemon():
+        server_name = "libvirtd"
+    return server_name
+
+
+def managed_daemon_config(conf_type="virtproxyd"):
+    """
+    Determine different daemon config under different daemon mode.
+    """
```

Comment:
Would you please update docstring for parameter and return value?

Reply:
@Yingshun updated